api/src/main/java/org/openmrs/module/openconceptlab/Subscription.java

```diff
@@ -86,11 +94,11 @@ public boolean isManual() {
 		return (days == null && hours == null && minutes == null) || (days == 0 && hours == 0 && minutes == 0);
 	}
 
-	public boolean isSubscribedToSnapshot() {
+	public Boolean getSubscribedToSnapshot() {
```

Comment:
Didn't I say to use "boolean isSubscribedToSnapshot()"? It's better as you don't have to check for null, ever! :)